cluster/cluster.yaml

```diff
@@ -1614,6 +1614,47 @@ Resources:
             Action:
             - "kms:Encrypt"
             - "kms:Decrypt"
+          - Sid: "Deny PowerUser to manage or use this key"
+            Effect: "Deny"
+            Principal:
+              AWS: "arn:aws:iam::{{ .Cluster.InfrastructureAccount | getAWSAccountID }}:role/Shibboleth-PowerUser"
```

Comment:
Why not just Allow only Administrator e.g. white list? This currently wouldn't deny `Shibboleth-Manual` for instance.

Reply:
Thanks for the hint. I didn't think of `Manual` :disappointed: 

The problem is that PowerUser gets `Allow: kms:*` from the IAM policy and I don't intend to change that just yet. That leaves the black list like that.

There's a chance to define a white list here as well and have the final permissions be the intersection of both but I'm not sure. I'll check that but if you have any hints I'm happy to hear them.

Reply:
@mikkeloscar This works, thanks!